src/main/scala/org/constellation/domain/transaction/TransactionChainService.scala

```diff
@@ -0,0 +1,44 @@
+package org.constellation.domain.transaction
+
+import cats.effect.Concurrent
+import cats.implicits._
+import io.chrisdavenport.log4cats.Logger
+import org.constellation.primitives.concurrency.SingleRef
+
+class TransactionChainService[F[_]: Concurrent: Logger] {
```

Comment:
There is no logger usage, if yoi plan to add so please create one directly in class it's then easier to filter out specific domains by setting package levels of logger e.g  `<logger name="org.constellation.domain.transaction" level="DEBUG">`

Reply:
:+1: